node/runtime/src/lib.rs

```diff
@@ -487,6 +487,7 @@ construct_runtime!(
 		NodeBlock = node_primitives::Block,
 		UncheckedExtrinsic = UncheckedExtrinsic
 	{
+		Randomness: randomness::{Module, Call, Storage},
```

Comment:
new module should be added to end to avoid breaking the module index, which is used to construct transactions